docs/_docs/triggers.md

```diff
@@ -87,42 +87,58 @@ Alternately, if your app supports REST Hooks—webhook subscriptions that can be
 
 ![](https://zappy.zapier.com/CF1A11AF-949A-4C74-AFCD-37F4F4C5B362.png)
 
-**Unsubscribe**
+We recommend that a successful Subscribe response return a 201. The data returned should include any data needed to attempt a later Unsubscribe request. This data will be stored in `bundle.subscribeData`.
 
-When Zapier sends the request to your API to unsubscribe the webhook, it includes the the unique ID that was returned during the Subscribe request, and you can reference it in the unsubscribe request, on the `bundle.subscribeData` object.  
+#### Unsubscribe
+
+When Zapier sends the request to your API to unsubscribe the webhook, it can reference any data that was returned during the Subscribe request and stored in `bundle.subscribeData`.
 
 ![](https://zappy.zapier.com/EF2923B4-B361-45A6-983C-BA2A57DC5623.png)
 
-**Perform List**
+#### Perform List
 
 In addition to the Subscribe and Unsubscribe requests, it's important to add a Perform List request where Zapier can check for recent items. This will be used to fetch data when users are setting up and testing the Zap. If you don't define a Perform List, then the user needs to go into your app and do something to generate a new event while the Zap editor waits for data, which is not an optimal experience.
 
 <a id="perform"></a>
-**Perform**
+#### Perform
 
 ![REST Hook Perform](https://cdn.zapier.com/storage/photos/5c6c154be13fbcad2a5dc8cb786f2dea.png)
 
-Finally, in the Perform, you can customize the code to evaluate the data your app's webhooks pass to Zapier. By default, Zapier includes `return [bundle.cleanedRequest];` to return the object from the webhook. If your data needs to be transformed, or includes multiple objects, add custom code to parse the response data in `bundle.cleanedRequest` within the Perform and turn it into an array of objects, such as [this example code](https://github.com/zapier/zapier-platform/blob/master/example-apps/rest-hooks/triggers/recipe.js#L42).
+Finally, in the Perform, you can customize the code to evaluate the data your app's webhooks pass to Zapier. By default, Zapier includes `return [bundle.cleanedRequest];` to return the request data from the webhook as an array. If your data needs to be transformed, or includes multiple objects, add custom code to parse the response data in `bundle.cleanedRequest` within the Perform and turn it into an array of objects, such as [this example code](https://github.com/zapier/zapier-platform/blob/master/example-apps/rest-hooks/triggers/recipe.js#L42).
+
+Both the Perform and Perform List methods should return arrays, even if they only contain one object. The default Perform code adds an array around the cleaned webhook payload, so if your webhook already provides an array, you can remove this additional array and simply `return bundle.cleanedRequest;`.
```

Comment:
I'd rephrase:

```suggestion
Both the Perform and Perform List methods should return an array, even if they only contain one object. The default Perform code adds an array around the cleaned webhook payload, as in `return [bundle.cleanedRequest];`, so if your webhook already provides an array, you can simply `return bundle.cleanedRequest;`.
```

Reply:
You're right that arrays isn't clear here, but there are numeric agreement issues with `an array` with the rest of the sentence, and the default code is mentioned in the previous paragraph, so I may make a slightly different adjustment on this.